spec.html

```diff
@@ -27994,9 +27994,9 @@ <h1>Year Number</h1>
         <emu-eqn id="eqn-TimeFromYear" aoid="TimeFromYear">TimeFromYear(_y_) = msPerDay &times; DayFromYear(_y_)</emu-eqn>
         <p>A time value determines a year by:</p>
         <emu-eqn id="eqn-YearFromTime" aoid="YearFromTime">YearFromTime(_t_) = the largest integer _y_ (closest to positive infinity) such that TimeFromYear(_y_) &le; _t_</emu-eqn>
-        <p>The leap-year function is 1 for a time within a leap year and otherwise is zero:</p>
+        <p>The leap-year function is 1 for a time within a leap year and otherwise is *+0*:</p>
         <emu-eqn id="eqn-InLeapYear" aoid="InLeapYear">InLeapYear(_t_)
-          = 0 if DaysInYear(YearFromTime(_t_)) = 365
+          = *+0* if DaysInYear(YearFromTime(_t_)) = 365
           = 1 if DaysInYear(YearFromTime(_t_)) = 366
```

Comment:
I'd suggest putting the two occurrences of "1" in asterisks as well, for balance.